### PR TITLE
Add progress bar for the EDA calculation in spatial LDA

### PR DESCRIPTION
**Pull request by alex-l-kong**
**What is the purpose of this PR?**

Begins addressing #566. Our repo includes a lot of intensive functions, so we'll need to ensure the user receives progress updates for these so they don't get left in the dark.

**How did you implement your changes**

Use the `tqdm` function in the `tqdm.notebook` function.

Based on our research and discussion with @ackagel, the only function that needs this for now is `spLDA.preprocessing.compute_topic_modeling`. There are various reasons others don't need them:

- They already have a progress bar or print outs per every `n` iterations
- They are direct calls to an external package, which we can't wrap easily in `tqdm` short of making a contribution to said package
- In the case of neighborhood analysis, progress will be delayed until after the MIBI workshop

### CODE REVIEW

**Comment by alex-l-kong** [comment]
Also needed to update the `start_docker.sh` script, since it was attempting to pull a non-existant `spatial-ark-test` image. Added the `site-packages` reference back for the time being too.

**Review by ngreenwald — CHANGES_REQUESTED** [review]
You'll need to coordinate with Adam about the plan for the local version of `ark`

**Comment by alex-l-kong** [comment]
@ngreenwald just updated `start_docker.sh` to reflect @ackagel's work in `main`.

**Comment by alex-l-kong** [comment]
@ngreenwald just updated `start_docker.sh` to reflect @ackagel's work in `main`.

File: ark/spLDA/processing.py [hunk 1]
```diff
@@ -278,7 +279,8 @@ def compute_topic_eda(features, featurization, topics, silhouette=False, num_boo
     stats = dict(zip(stat_names, [{} for name in stat_names]))
 
     # iterative over topic number candidates
-    for k in topics:
+    pb_format = '{l_bar}{bar}| {n_fmt}/{total_fmt} [{elapsed}<{remaining}]'
+    for k in tqdm(topics, bar_format=pb_format):
```
Comment (ngreenwald) [inline ark/spLDA/processing.py:283]: Did you run the notebook to make sure the display looks as expected?

**Review by ngreenwald — CHANGES_REQUESTED** [review]

File: ark/spLDA/processing.py (hunk 1 above)
Comment (alex-l-kong) [reply ark/spLDA/processing.py:283]: It does.

**Pull request merged**